futures/README.md

```diff
@@ -88,7 +88,7 @@ fn main() {
       // dispatching events on time.
       // If we ran it as part of the "main" chain of futures, we might end up not sending
       // some heartbeats if we don't poll often enough (because of some blocking task or such).
-      tokio::spawn(heartbeat.map_err(|_| ()));
+      tokio::spawn(heartbeat(&client).map_err(|_| ()));
```

Comment:
The docs for Client::connect say that the heartbeat function needs to be passed a client. This is the error I get when I don't pass it a client (this builds for me for some reason):
![image](https://user-images.githubusercontent.com/14206675/39585710-ee92d734-4ee4-11e8-94a8-9c20b51b1827.png)

And sorry if I'm missing anything, I'm rather new to rust.

Reply:
You're reading the documentation from the latest stable release, you're testing with the latest stable release, but you're trying to build an example from the current development tree.
The way we handle heartbeat has changed since the last released.
Your code builds with the last stable version. The current code works with the current development version.